File: templates/CSharp/Procon.cs
using System;
using System.IO;


class MainClass
{
    private static void Main()
    {
        Console.SetOut(new StreamWriter(Console.OpenStandardOutput()) { AutoFlush = false });

        string line;
        while ((line = Console.ReadLine()) != null)
        {
            var tokens = line.Split(' ');
            <+CURSOR+>
        }

        Console.Out.Flush();
    }
}

File: templates/JavaScript/procon.js
(function(main) {
  'use strict';

  if (typeof process !== 'undefined') {
    process.stdin.resume();
    process.stdin.setEncoding('utf8');

    var lines = [];
    require('readline').createInterface({
      input: process.stdin,
      output: process.stdout
    }).on('line', function(line) {
      lines.push(line);
    });
    process.stdin.on('end', function() {
      var lineCnt = 0;
      var nLine = lines.length;
      main(console.log, function() {
        return lineCnt < nLine ? lines[lineCnt++] : null;
      });
    });
  } else {
    main(print, readline);
  }
})(function(print, readline) {
  'use strict';

  var line;
  while ((line = readline()) !== null) {
    print(line.split(' ').map(Number));
    <+CURSOR+>
  }
});
